src/main/kotlin/lotto/exceptions/LottoException.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.exceptions
+
+import lotto.domain.LottoNumber
+import lotto.domain.LottoTicket
+
+sealed class LottoException(msg: String) : IllegalArgumentException("[ERROR]: $msg") {
+    class InvalidAmountFormatException(input: String) : LottoException("Wrong amount format: $input.\n")
+
+    class InvalidAmountException(amount: Int) :
+        LottoException(
+            "Wrong amount: $amount, " +
+                    "Amount should be at least ${LottoTicket.PRICE_OF_TICKET} ${LottoTicket.CURRENCY}" +
+                    "and divisible by ${LottoTicket.PRICE_OF_TICKET}.\n",
+        )
+
+    class InvalidWinningNumbersFormatException(input: String) :
+        LottoException("Invalid winning numbers format: '$input'. Expected numbers separated by commas.")
+
+    class InvalidWinningNumbersException() :
+        LottoException(
+            "Invalid winning numbers: Expected ${LottoTicket.SUFFICIENT_SIZE} numbers in range + " +
+                    "${LottoNumber.MIN_RANGE_NUMBER} to ${LottoNumber.MAX_RANGE_NUMBER}\n",
+        )
```

Comment:
Are these all in use right now? 🤔 

Plz remove the unused code.
Fellow developer will feel confused!